docs/getting_started.md

```diff
@@ -156,32 +193,78 @@ Encrypting a message with a symmetric cipher:
 1. Call `psa_cipher_update` one or more times, passing either the whole or only a fragment of the message each time.
 1. Call `psa_cipher_finish` to end the operation and output the encrypted message.
 
-Encrypting random data using an AES key in cipher block chain (CBC) mode with no padding (assuming all prerequisites have been fulfilled):
+Encrypting some data using an AES key in cipher block chain (CBC) mode with no padding (assuming all prerequisites have been fulfilled):
```

Comment:
"Some" sounds a bit funny to me, especially in docs - maybe it should just be "Encrypt data" (here and in the source code)?